linera-core/src/client/mod.rs

```diff
@@ -164,6 +165,9 @@ mod metrics {
     });
 }
 
+/// The number of chain workers that can be in memory at the same time.
+const CHAIN_WORKER_LIMIT: usize = 20;
```

Comment:
Whenever, we have such limits, which are an unavoidable fact of programming, we would like a little bit more info:
* What other constants and latency are related.
* What aspects to consider. I guess, here it is memory. Something else?

Reply:
I expanded the comment a bit. No, I think you're right, it's just a tradeoff between performance and memory usage.

(Note that I didn't introduce this value in this PR; it was already hard-coded.)